docs/containers/ssh.md

```diff
@@ -0,0 +1,54 @@
+---
+Area: containers
+ContentId: DDE07043-BA8C-4D75-B392-ABACC31F6EA8
+PageTitle: Connect to Docker running on a remote machine
+DateApproved: 
+MetaDescription: Connect to a container image running on a remote machine, using Visual Studio Code.
+---
+# Connect to a remote Docker daemon over SSH
+
+## Overview
+
+In order to connect to a remote Docker daemon over SSH (as opposed to HTTPS with certificate authentication), there are two options for configuring the extension.
+
+## Using Visual Studio Code Remoting
+
+The simplest way is to use VS Code's [Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extension, from the [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) extension pack.
+
+1. Run command `Remote-SSH: Add new SSH host...` and follow the prompts to set up a connection to the target host.
+
+1. Run command `Remote-SSH: Connect to host...` and connect to the host.
+
+1. A new VS Code window opens, remoted to the target machine. If using password authentication, the password will be prompted here. It is recommended to set up [SSH key authentication](https://www.ssh.com/ssh/public-key-authentication), for ease of use. In the Extensions tab, install the Docker extension (on the remote host) (a reload may be required after this step):
+
+![image](https://user-images.githubusercontent.com/36966225/66958480-3f280b80-f036-11e9-8d75-b4e55eb3913f.png)
```

Comment:
Add local version of the image
Also need descriptive alt-text for image with users with screen readers.